include/ck_tile/ops/epilogue/cshuffle_epilogue.hpp

```diff
@@ -41,6 +41,12 @@ struct CShuffleEpilogueProblem
     static constexpr index_t kKPerXdl                      = kKPerXdl_;
     static constexpr index_t isCTransposed                 = isCTransposed_;
     static constexpr memory_operation_enum MemoryOperation = MemoryOperation_;
+    // NumMXdlPerWavePerShuffle used for in each shuffle iteration how many xdl tiles in M
+    // dimensions per wave
+    static constexpr index_t NumMXdlPerWavePerShuffle = 1;
```

Comment:
I don't like this parameter has hardcoded value. It would be best if there's a way to calculate its optimal value in order to maximize vector size. Otherwise please expose it through template parameter.